scripts/build_nmd_file.py

```diff
@@ -18,6 +18,10 @@
 structure = parsePDB(pdb_file)
 
 ensemble.setCoords(structure)
+
+# Limit to alpha carbons to keep a low memory profile
+structure = structure.select('calpha')
```

Comment:
The spike protein from the structural bio exercises is fairly large. On my laptop, even reducing just to protein atoms results in an error that the program can't allocate about a hundred gigabytes of memory. (On the VSC, it's just quietly killed)

This seems to work to generate an NMD file that I could open in VMD.